test/test_xla_virtual_device.py

```diff
@@ -0,0 +1,90 @@
+import os
```

Comment:
I think we need to update run_test.sh. Can we also move all SPMD test into a separate dir like `test/spmd/`